supervisor/store/data.py

```diff
@@ -86,15 +86,14 @@ def _read_git_repository(self, path: Path) -> None:
         self.repositories[slug] = repository_info
         self._read_addons_folder(path, slug)
 
-    def _read_addons_folder(self, path: Path, repository: Dict) -> None:
-        """Read data from add-ons folder."""
+    def _find_addons(self, path: Path, repository: Dict) -> Optional[list[Path]]:
+        """Find add-ons in the path."""
         try:
             # Generate a list without artefact, safe for corruptions
             addon_list = [
                 addon
                 for addon in path.glob("**/config.*")
-                if ".git" not in addon.parts
-                and ".github" not in addon.parts
+                if not str(addon).startswith(".")
```

Comment:
We using parts because that can be somewhere in the path. I think they generate a full name "/data/addons/git/123231/..." which will now not work anymore?

Reply:
.git should be at the root of the repository